sway-fmt-v2/src/utils/path.rs

```diff
@@ -0,0 +1,71 @@
+use crate::fmt::*;
+use std::fmt::Write;
+use sway_parse::{PathType, PathTypeSegment, QualifiedPathRoot};
+use sway_types::Spanned;
+
+impl Format for QualifiedPathRoot {
+    fn format(
+        &self,
+        formatted_code: &mut FormattedCode,
+        formatter: &mut Formatter,
+    ) -> Result<(), FormatterError> {
+        // Commenting this out since this will err until `Ty` formatting gets in
+        // self.ty.format(formatted_code, formatter)?;
```

Comment:
Should this be commented out? The comment mentions we can uncomment when `Ty` formatting lands, but it seems like that's happening as a part of this PR? Does this get tested yet?

Reply:
I am uncommenting it but for actually testing this I think we need something like the following to the existing test which requires us to have the baseline for the `fn` formatting (which is #2173):

```rust
fn test_path_ty() { 
    match TestTy {
        TestTy::String(str) = > {},
         _ => {}
    }
}
```
I couldn't come up with another way to test that part and if we cannot find another way a test for this can be added with #2173 like above.